core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -701,7 +720,7 @@ private void joinQuorum() {
             .setHost(localAddress.getHostString())
             .setRpcPort(localAddress.getPort()))
         .build();
-    RaftClient client = createClient();
+    RaftClient client = createClient(false);
```

Comment:
Is this `false` because `getCurrentGroup` will not work until joining?

Reply:
Yes. The API call will not work before cluster is up and thus a waste of time during startup.